### ISSUE TITLE
[Spike] Investigate A11Y Analysis for Each Step in a Feature Test

### ISSUE THREAD

**Issue by Andrei15193** [Andrei15193/HintKeep#35]
Based on #34, HTML snapshots can be extracted from JSDom through the `serialize` method. This allows for multiple snapshots to be extracted while running a feature test, potentially after each step using the `AfterStep` hook.

Investigate a11y tools for analysing HTML files and generating reports. Ideally with CI/CD integration, an optional flag should be set when a11y analysis should be done. This may be more relevant to do during CI builds rather than locally although running an analysis on local files should be possible as well.

Useful links:
* https://www.w3.org/WAI/test-evaluate/tools/list/
* https://github.com/be-lenka/be-a11y

**Comment by Andrei15193** [comment]
This has been completed, a11y checks have been added to the demo feature and are now included for each build.
